pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -7,10 +7,10 @@
         >>
   %div.announce-content
     %div.content-img-container
-      %img.content-img{:src=>"/images/mc/fit-x108/MC_Box_V3.jpg"}
+      %img.content-img{:src=>"/images/fit-x108/volunteer-engineer-thumbnail.jpeg"}
```

Comment:
Can you use this image (copy to code.org/images)?

https://hourofcode.com/us/images/fit-300/volunteer-map.png